processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -75,6 +75,8 @@
   public static final byte LONG_LAST_CACHE_TYPE_ID = 0x18;
   public static final byte TIMESTAMP_CACHE_TYPE_ID = 0x19;
   public static final byte VARIANCE_CACHE_TYPE_ID = 0x1A;
+  public static final byte ACCURATE_CARDINALITY_CACHE_TYPE_ID = 0x1B;
```

Comment:
These cache id's are already in use, they all need to be distinct. Hopefully soon we can create a mechanism to make this a less manual process with #6823, but for now we need to keep this correct by hand. `0x35` is the current highest, but there [is another aggregator PR that is using up to 0x37](https://github.com/apache/incubator-druid/pull/6581/files#diff-91373812bf892462c351704c53c25ff7R122) so you should probably just start with `0x38`.